plugin/route53/setup.go

```diff
@@ -46,7 +62,8 @@ func setup(c *caddy.Controller) error {
 		var providers []credentials.Provider
 		var fall fall.F
 
-		refresh := time.Duration(1) * time.Minute // default update frequency to 1 minute
+		refresh := time.Duration(1) * time.Minute       // default update frequency to 1 minute
```

Comment:
You don't need to cast this to `time.Duration`. In fact this makes it less readable.

Reply:
It's the original code, it wasn't really changed by me, I just formatted it

Reply:
https://github.com/coredns/coredns/blob/master/plugin/route53/setup.go#L49

Reply:
🤔 did i write it this way? sorry, nvm then, i'll fix it later

Reply:
I changed it.